core/src/test/java/org/commonjava/indy/fixture/MockContentAdvisor.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (C) 2013 Red Hat, Inc. (jdcasey@commonjava.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.fixture;
+
+import org.commonjava.indy.spi.pkg.ContentAdvisor;
+import org.commonjava.indy.spi.pkg.ContentQuality;
+import org.commonjava.maven.atlas.ident.util.ArtifactPathInfo;
+
+import javax.enterprise.inject.Default;
+
+//TODO: This provider is fully duplicated with org.commonjava.indy.test.fixture.core.MockContentAdvisor in
+//      test/fixture-core for some dependency reasons in addons. Should be refactored in another common
+//      test module in the future.
+
+@Default
+public class MockContentAdvisor
+        implements ContentAdvisor
+{
+    @Override
```

Comment:
It might be worthwhile to have a field here with a pre-set ContentQuality, or a mapping of path -> ContentQuality. That would reduce the likelihood that some problem related to ArtifactPathInfo will lead to this mock breaking. The test could then register the paths it's interested in with appropriate content quality, as a way of setting its initial conditions. It would also allow new tests that need to verify metadata quality levels without modification.

Dunno, just a thought.